demo.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>This is a paragraph.</p>
```

Comment:
Please remove duplicate lines.

Reply:
this is for example

Reply:
this is for example

Reply:
Even though it's a example code. Kindly remove duplicate lines of code.

Reply:
removing duplicate lines